contrib/examples/actions/chains/json_escape_jinja_filter.yaml

```diff
@@ -0,0 +1,11 @@
+---
+vars:
+    escape_str: 'Random text from another action output """ which will break JSON and needs escaping'
+
+chain:
+  -
+     name: "action1"
+     ref: "core.local"
+     parameters:
+        cmd: |
+          echo -E '[{"title":{{ escape_str | json_escape }}, "color": "#37C200"}]'
```

Comment:
Since I'm now using `json.dumps` I think the input may need to be JSON, so that example wouldn't work. Let me tinker a bit more